test/distro/distro.spec.js

```diff
@@ -7,6 +7,13 @@ const fs = require('fs'),
 
 const DIST_DIR = path.join(__dirname, '../../dist');
 
+const EXPORTS = [
+  'bpmn-js-element-templates',
+  'bpmn-js-element-templates/core',
+  'bpmn-js-element-templates/dist/assets/element-templates.css',
+  'bpmn-js-element-templates/dist/bpmn-js-element-templates.umd.js'
```

Comment:
We can consider to drop this UMD distribution with the next major (or not actually export).

Reply:
Agreed.